docker-compose.yml

```diff
@@ -19,6 +20,15 @@ web:
     - MYSQL_ROOT_PASSWORD='docker'
     - MYSQL_DATABASE='olympia'
     - PYTHONDONTWRITEBYTECODE=1
+    - BROKER_URL=amqp://olympia:olympia@rabbitmq/olympia
+
+worker:
+  build: .
```

Comment:
Might be cleaner to use `extends` here. Or define this as the base, and have `web` extend it, since that adds exposed ports.